test/client/fixtures/sandbox/code-instrumentation/getters-test.js

```diff
@@ -524,3 +524,48 @@ test('we should clean up html and remove extra namespaces from svg (GH-1083)', f
 
     strictEqual(getProperty(div, 'innerHTML'), nativeDiv.innerHTML);
 });
+
+test('we should not process element\'s properties if they do not exist (GH-1164)', function () {
+    var svg             = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
+    var html            = '<a href="/path">link</a>';
+    var processedHtml   = processHtml(html);
+    var processedStyle  = styleProcessor.process('body {}', null, true);
+    var htmlPproperties = ['innerHTML', 'outerHTML'];
+    var textProperties  = ['innerText', 'text', 'textContent'];
+    var haveAsserts     = false;
+
+    htmlPproperties.forEach(function (property) {
+        var svgHasProperty = svg[property] !== void 0;
```

Comment:
Let's change test scenario:
1. when element has target property we will check that getter returns processed value
2. when element doesn't have target property we check that getter return `void 0` on first check and set value on second check